### PR TITLE
boost-bcp boost-build boost-mpi boost-python3 boost 1.81.0

### PR DESCRIPTION
**Pull request by fanquake**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
@fanquake can you split the PR into five commits?
- boost 1.81.0 (including alias update)
- boost-bcp 1.81.0
- boost-build 1.81.0
- boost-mpi 1.81.0
- boost-python3 1.81.0

**Comment by fanquake** [comment]
> can you split the PR into five commits?

Sure. Split into 5.

**Comment by chenrui333** [comment]
Thanks!!

File: Formula/boost-python3.rb [hunk 1]
```diff
@@ -6,15 +6,8 @@ class BoostPython3 < Formula
   head "https://github.com/boostorg/boost.git", branch: "master"
 
   stable do
```
Comment (cho-m) [inline Formula/boost-python3.rb:8]: Remove the `revision 1` above these lines. Also should get rid of `stable do` block and move `url`/`sha256` after the `homepage`.

**Comment by fxcoudert** [comment]
Revision bumps needed for:

```
brew test --retry --verbose openimageio
brew test --retry --verbose pinocchio
brew test --retry --verbose c10t
brew test --retry --verbose collada-dom
brew test --retry --verbose fbthrift
brew test --retry --verbose folly
brew test --retry --verbose fuego
brew test --retry --verbose gource
brew test --retry --verbose innoextract
brew test --retry --verbose ledger
brew test --retry --verbose mapcrafter
brew test --retry --verbose monero
brew test --retry --verbose pcl
brew test --retry --verbose supertux
brew test --retry --verbose uhd
brew test --retry --verbose vineyard
brew test --retry --verbose vtk
brew test --retry --verbose wownero
```

**Comment by fanquake** [comment]
> Revision bumps needed for:

Added revision bumps.

**Comment by chenrui333** [comment]
let's see how it goes this time.

**Comment by chenrui333** [comment]
cannot push update the branch, use [this PR](https://github.com/Homebrew/homebrew-core/pull/119529) instead.

**Comment by fanquake** [comment]
> cannot push update the branch

What do you mean, and why is that necessary?

**Comment by p-linnane** [comment]
When you opened this PR you chose to disallow maintainers from pushing updates to the branch. That prevents us from making any changes or doing testing and requires you to do everything. That limits our ability to triage, and try different things to get the PR merged.

**Pull request closed without merging**